backend/dataall/base/api/gql/graphql_argument.py

```diff
@@ -8,23 +8,29 @@
 
 
 class Argument:
-    def __init__(self, name, type):
+    def __init__(self, name, type, description=''):
         self.name = name
+        self.description = description
         if isinstance(get_named_type(type), (Scalar, InputType, Ref, Enum)):
             self.type = type  # get_named_type(type)
         else:
             raise Exception('Invalid Argument Type')
 
     def gql(self):
+        description_str = ''
+        n = '\n'
+        if self.description:
+            description_str = f' """{self.description}""" {n}'
+
         if isinstance(self.type, Enum):
-            return f'{self.name} : {self.type.name}'
+            return f'{description_str}{self.name} : {self.type.name}'
```

Comment:
Also, do we want to store \n in description itself? 
IMHO, it's better to format it in place
'{description_str}\n{self.name} : {self.type.name}'

Reply:
well we do not need `\n` if there is a description - so the new line is tied to the description a bit

I have not played around with how descriptions can be interpreted - I can play around a bit to see if can keep all on same line but if not I will keep it as is

Reply:
From my research - the standard seems to be adding comments on a new line above the type or argument you are describing (ex:  https://medium.com/@krishnaregmi/how-to-write-comments-in-a-graphql-schema-to-enhance-auto-generated-documentation-c0047125ea24)

I think best to follow the same